extensions-twitter/scribe-emitter/src/main/java/org/apache/druid/emitter/scribe/ScribeEmitter.java

```diff
@@ -80,32 +80,31 @@ public void emit(Event event)
       catch (JsonProcessingException e) {
         log.warn("" + e + " Could not process administrative string as JSON object " + event);
       }
-      catch (TException e) {
-        log.warn("" + e + ", Could not serialize thrift object of query: " + event);
+      catch (Exception e) {
+        log.warn("" + e + ", Could not emit event: " + event);
       }
     } else if (event instanceof ServiceMetricEvent && ((ServiceMetricEvent) event).getMetric().compareTo(INDEXING_CONFIG) == 0) {
       try {
         indexingLogScriber.scribe(ScribeIndexingLogEntry.createScribeIndexingLogEntry(event, scribeEmitterConfig).toThrift());
       }
-      catch (TException e) {
-        log.warn("" + e + ", Could not serialize thrift object of query: " + event);
+      catch (Exception e) {
+        log.warn("" + e + ", Could not emit event: " + event);
       }
     }
   }
 
   @Override
   public void flush()
   {
-    requestLogScriber.flush();
-    adminLogScriber.flush();
-    indexingLogScriber.flush();
   }
 
   @Override
   public void close()
   {
-    requestLogScriber.close();
-    adminLogScriber.close();
-    indexingLogScriber.close();
+  }
+
+  private boolean isDatacenterHostGCP(String datacenterHost)
+  {
+    return datacenterHost.compareTo(GCP) == 0;
```

Comment:
how about use equals?

Reply:
In this case it is fine to use either as the types are strictly defined to be String. But in general if you want string comparison it is better to use compareTo in my opinion because equals takes in a Object and you can pass in anything.